src/rust/lib.rs

```diff
@@ -5,6 +5,22 @@
 #![deny(clippy::all)]
 #![feature(maybe_uninit_uninit_array, new_uninit)]
 #![feature(try_blocks)]
+#![cfg_attr(feature = "strict", deny(clippy:all))]
```

Comment:
Just out of curiosity (this was also present in the old inetstack/src/lib.rs these were apparently pulled from) does this actually do anything different if feature "strict" is specified?  Since there is a non-qualified "deny(clippy:all)" up on line 5?

Reply:
This is redundant. I shall now get the integration cleaner. Thanks for noting this.